pylib/gyp/generator/make.py

```diff
@@ -2381,7 +2381,7 @@ def WriteAutoRegenerationRule(params, root_makefile, makefile_name, build_files)
         % {
             "makefile_name": makefile_name,
             "deps": replace_sep(
-                " ".join(SourceifyAndQuoteSpaces(bf) for bf in build_files)
+                " ".join(SourceifyAndQuoteSpaces(bf) for bf in sorted(build_files))
```

Comment:
```suggestion
                " ".join(sorted( SourceifyAndQuoteSpaces(bf) for bf in build_files))
```
Would `SourceifyAndQuoteSpaces(build_file)` change the sort order of the files?

Reply:
@cclauss 

Thanks for your review.

I don't think `SourceifyAndQuoteSpaces` would change the sort order. Instead, it just format the path as below.
https://github.com/nodejs/gyp-next/blob/e97fd2509a6fb6452eab5e9b17f4cae3fc49f25e/pylib/gyp/generator/make.py#L749-L764